trail/query/views.py

```diff
@@ -1,6 +1,42 @@
-from django.http import HttpResponse
+from django import forms
+# from django.forms import ModelForm
+from django.apps import apps
+from django.shortcuts import render
+
+
+Metadata = apps.get_model('upload', 'Metadata')
+# Needs to think about adding SELECT DISTINCT etc.
+
+
+class MetadataForm(forms.Form):
```

Comment:
Please add a docstring to this class. Again, I suggest following the guide at https://developer.lsst.io/python/numpydoc.html#documenting-classes